### ISSUE TITLE
No unit tests?

### ISSUE THREAD

**Issue by binarykitchen** [swannodette/mori#174]
Looks like there are none. Any reasons for that?

**Comment by neverfox** [comment]
Because there's not really any code. This is merely a facade over [cljs.core](https://github.com/clojure/clojurescript). The tests are there.

**Comment by binarykitchen** [comment]
Yes, aware of that. And what's the command you use to compile it into JS? Any chance to tell the compiler not to minify when doing so?

**Comment by neverfox** [comment]
If you changed to using `:optimizations :whitespace` in the `dev` cljsbuild profile in project.clj and run that from `scripts/build.sh`, you should get unminified code.

**Comment by binarykitchen** [comment]
any chance you can change to that? would be great, thanks

**Comment by neverfox** [comment]
I'm not the maintainer.

**Comment by binarykitchen** [comment]
@swannodette ping?
